src/gui/raster/qgsrasterlayerproperties.cpp

```diff
@@ -1582,6 +1602,43 @@ void QgsRasterLayerProperties::optionsStackedWidget_CurrentChanged( int index )
   }
 }
 
+void QgsRasterLayerProperties::initializeDataDefinedButton( QgsPropertyOverrideButton *button, QgsRasterPipe::Property key )
```

Comment:
Do you also need to add code to handle the styling panel?

Reply:
Can you clarify?

Reply:
(actually it's already handled in this pr)